data/hai/hai missions.txt

```diff
@@ -287,6 +288,7 @@ mission "Unwanted Cargo"
 	destination "Allhome"
 	to offer
 		has "event: hai in cargo"
+		not "event: hai secret leaks"
```

Comment:
This should also be on the trigger mission.

Reply:
Should it? Whether or not anything is "legal" shouldn't have any impact on a Hai child stowing away in the cargo hold.